bin/lookup.cmd

```diff
@@ -0,0 +1,36 @@
+::[Bat To Exe Converter]
```

Comment:
What does this file do?

Reply:
Just a lets Windows run the binary node call, I'm not sure if that header part is even required tbh, I lifted it from an OCLIF auto-generated cmd file, but looking at their source it doesn't seem needed:
https://github.com/oclif/oclif/blob/master/bin/run.cmd

Likely just an artifact of whatever tool generates the cmd file in their build process so can be removed.